{
  "language": {
    "name": "python",
    "engine": "CPython",
    "version": "3.8.19"
  },
  "client": {
    "name": "appmap",
    "url": "https://github.com/applandinc/appmap-python"
  },
  "git": {
    "repository": "https://github.com/land-of-apps/django-oscar.git",
    "branch": "master",
    "commit": "01aa57720e1e703bdc166b20350eff665be60dad"
  },
  "frameworks": [
    {
      "name": "pytest",
      "version": "7.4.4"
    },
    {
      "name": "Django",
      "version": "4.2.13"
    }
  ],
  "feature_group": "Order creator error cases",
  "recording": {
    "defined_class": "tests.integration.order.test_creator.TestOrderCreatorErrorCases",
    "method_id": "test_raises_exception_if_duplicate_order_number_passed"
  },
  "source_location": "tests/integration/order/test_creator.py:60",
  "name": "Order creator error cases raises exception if duplicate order number passed",
  "feature": "Raises exception if duplicate order number passed",
  "app": "django-oscar",
  "recorder": {
    "name": "pytest",
    "type": "tests"
  },
  "test_status": "succeeded"
}